<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>649</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Edwardian Script ITC</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">#frame{
background-color:rgb(255, 255, 255);
border-radius:5px;
}

QToolButton{
border:none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icon/close.png</normaloff>icon/close.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QToolButton" name="toolButton_5">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>130</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icon.png</normaloff>icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI Variable Small Semibol</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Welcome</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>600</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>15</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>400</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
background-color:rgba(255, 255, 255,0);
border:none;
border-bottom:1px solid black;
}
QPushButton{
background-color:rgb(0,0,0);
color:rgb(255,255,255);
border-radius:10px
}

QPushButton:pressed{
padding-top:5px;
padding-left:5px;
}</string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="Page_Login">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QLineEdit" name="Login_account">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="placeholderText">
                    <string>帳號</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="Login_password">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>密碼</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="Login_correct">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>登入</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="Page_forget_password">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLineEdit" name="forget_password_2">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="placeholderText">
                    <string>帳號</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="forget_account">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>更換密碼</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="forget_password_1">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>確認密碼</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="forget_correct">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>變更密碼</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="Page_Register">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLineEdit" name="register_account">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="placeholderText">
                    <string>帳號</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="register_password_1">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>密碼</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="register_password_2">
                   <property name="minimumSize">
                    <size>
                     <width>35</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                   </property>
                   <property name="placeholderText">
                    <string>確認密碼</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="register_correct">
                   <property name="minimumSize">
                    <size>
                     <width>20</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微軟正黑體</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>註冊</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="Login_button">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icon/login.png</normaloff>icon/login.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="Register_button">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icon/person_add.png</normaloff>icon/person_add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="password_button">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icon/password.png</normaloff>icon/password.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <widget class="QWidget" name="page"/>
            <widget class="QWidget" name="page_2"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
